substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVConstants.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, Red Hat Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.oracle.objectfile.pecoff.cv;
+
+public abstract class CVConstants {
+
+    /* names of relevant CodeView sections */
+    static final String CV_SYMBOL_SECTION_NAME = ".debug$S";
+    static final String CV_TYPE_SECTION_NAME = ".debug$T";
+    // static final String CV_RDATA_SECTION_NAME = ".rdata";
+    // static final String CV_PDATA_SECTION_NAME = ".pdata";
+    // static final String CV_XDATA_SECTION_NAME = ".xdata";
+    // static final String TEXT_SECTION_NAME = ".text";
+    // static final String DATA_SECTION_NAME = ".data";
+
+    /* CodeView section header signature */
+    static final int CV_SIGNATURE_C13 = 4;
+
+    /*
+     * Knobs
+     *
+     * (some may become Graal options in the future)
+     */
+
+    /*
```

Comment:
I'm not clear why you need JDK_SOURCE_BASE or GRAAL_SOURCE_BASE. They are not used. Will they be needed?

Reply:
They were temporary hooks that could have been turned into svm options; they predate the SourceCache.  Gone.